x-pack/plugins/licensing/common/types.ts

```diff
@@ -17,7 +17,8 @@ export enum LICENSE_TYPE {
   standard = 20,
```

Comment:
What is standard used for? Never seen that one.

Reply:
It is a cloud only license level (adds a few things on top of basic that are needed for ESS).

Reply:
Got it, thanks Peter.